### PR TITLE
B 19825 rename qaecsr to qae int

### PR DESCRIPTION
**Pull request by cameroncaci**
> [!WARNING]
> This branch and the main branch will have many differences due to a plethora of merge conflicts due to the "CSR" and "HQ" roles being added to int at the same time with the "QAE/CSR" rename following right behind

## Preivous INT PR
This is a new PR due to the last integration branch having too many discrepancies with the main version of the branch, despite it being brought over via a merge instead of cherry-picking. Old pr https://github.com/transcom/mymove/pull/12920

## Summary

Renames code-based and ui-base mentioned of QAE/CSR to just QAE (Acronym spelled out respectively). I left a few QAE/CSR combinations because they were respective of both QAE and CSR both being part of the commented code, or move search.

## Screenshots
image
image
image

### CODE REVIEW

**Comment by TevinAdams** [comment]
@cameroncaci  I still see qaecsr in the same name. For example "qaeCsrSearchForAndNavigateToMove()" inplaywright/tests/office/qaecsr/csrFlows.spec.js and "qaeCSRRoutes" insrc/constants/routes.js.  "QAECSRMoveSearch" insrc/pages/Office/QAECSRMoveSearch/QAECSRMoveSearch.test.jsx. I see that there are still QAECSR combination in names, texts and such. I know it may be more work changing those but should we go ahead and do that to minimize any confusion?

**Review by traskowskycaci — CHANGES_REQUESTED** [review]
Functionality looks good!

Left a comment on one file asking if you think we should break up qaeCsrSearchForAndNavigateToMove() into two functions - I kinda think so but let me know if you disagree.

The other instances of qaeCsr I see from searching, like qaeCSRRoutes, I'm fine with since I saw a comment from you saying QAEs and CSRs share routing but not permissions. Looks mostly good!

File: playwright/tests/utils/office/officeTest.js
[the remark this answers is not in the document]
Comment (cameroncaci) [reply playwright/tests/utils/office/officeTest.js:1]: I split this func in half with the latest commit, it makes sense to go ahead and separate this one. Resolved

**Comment by cameroncaci** [comment]
> @cameroncaci I still see qaecsr in the same name. For example "qaeCsrSearchForAndNavigateToMove()" inplaywright/tests/office/qaecsr/csrFlows.spec.js and "qaeCSRRoutes" insrc/constants/routes.js. "QAECSRMoveSearch" insrc/pages/Office/QAECSRMoveSearch/QAECSRMoveSearch.test.jsx. I see that there are still QAECSR combination in names, texts and such. I know it may be more work changing those but should we go ahead and do that to minimize any confusion?

Split the playwright funcs in half, requesting re-review

**Comment by cameroncaci** [comment]
Flaky tests failing

**Review by traskowskycaci — APPROVED** [review]
Looks SO nice! :rocket:

**Review by danieljordan-caci — APPROVED** [review]
Couldn't find anything that jumped out of me. Ran through flows of creating a QAE user as well as CSR and all seems to be working fine. LGTM!

**Review by deandreJones — APPROVED** [review]

**Pull request merged**